libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/couchdb/AttachmentConnector.java

```diff
@@ -75,26 +77,37 @@ public void deleteAttachment(String id) {
     }
 
     public void deleteAttachments(Collection<Attachment> attachments) {
-        Set<String> attachmentContentIds = getAttachmentContenIds(attachments);
+        Set<String> attachmentContentIds = getAttachmentContentIds(attachments);
         deleteAttachmentsByIds(attachmentContentIds);
     }
 
     private void deleteAttachmentsByIds(Collection<String> attachmentContentIds) {
         connector.deleteIds(attachmentContentIds, AttachmentContent.class);
     }
 
-    private Set<String> getAttachmentContenIds(Collection<Attachment> attachments) {
+    private Set<String> getAttachmentContentIds(Collection<Attachment> attachments) {
         return nullToEmptyCollection(attachments).stream()
                 .map(Attachment::getAttachmentContentId)
                 .collect(Collectors.toSet());
     }
 
-    public void deleteAttachmentDifference(Set<Attachment> before, Set<Attachment> after){
+    public void deleteAttachmentDifference(Set<Attachment> attachmentBefore, Set<Attachment> attachmentAfter){
```

Comment:
I don't mind the renaming, but then it should be plural, because these are sets. E.g. `attachmentsBefore`, `attachmentsAfter`.